oval_graph/client.py

```diff
@@ -74,13 +74,15 @@ def get_list_of_matched_rules(self):
     def get_list_of_lines(self):
         lines = ['== The Rule IDs ==']
         for rule in self.get_list_of_matched_rules():
-            lines.append("'" + rule + r'\b' + "'")
+            lines.append("^" + rule + "$")
         if self.show_not_selected_rules:
             for line in self.get_lines_of_wanted_not_selected_rules():
                 lines.append(line)
         lines.append(
             "You haven't got installed inquirer lib. "
-            "Please copy id rule with you want use and put it in command")
+            "Please copy id rule with you want use and"
+            " put it in command with regex characters "
+            "(^$). Or use arguments --all or --all-in-one.")
```

Comment:
You have a typo there (with), and the rest of the sentence could be improved as well. Try to give it one more shot. Moreover:

- The user is told that they don't have inquirer installed, but they don't know how that would help them. I would use something like "Interactive rule selection is not available, because inquirer is not installed".